src/main/resources/templates/sql-queries

```diff
@@ -9,7 +9,7 @@ CREATE TABLE car (
                     price decimal(10,3)
                     );
 
-INSERT INTO car (brand, model, vin, date_of_manufactured, color, price)
+INSERT INTO cars (brand, model, vin, date_of_manufactured, color, price)
 		VALUES ('Ferrari', '599 GTB', '1FTHF25M5RNB23608', '2011-07-09', 'Green', 22500.00),
```

Comment:
Why do we need values after the table is created?
Usually, hardcoded values are not used. If you need to populate the database with dummy values, you can have an integration test that does just that. For the integration tests we can use [MockMvc from Spring](https://www.baeldung.com/spring-boot-testing).

In the end, it's up to you. If needed, hardcoded values can be used as dummy data.

You will end up having APIs for all the CRUD operations, this is why I think you will not need some data to be present in the database when the application is started.